src/styles/global.css

```
@@ -2678,6 +2678,56 @@ a.tag:hover {
+.panel--nudging {
+  animation: panel-pulse 700ms ease-in-out;
+  will-change: filter;
+}
+
+@media (prefers-reduced-motion: reduce) {
+  .panel--settling,
+  .panel--nudging {
+    animation: none;
+  }
+  .panel.has-static-hint .panel-label {
+    border-bottom: 1px solid currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)